### PR TITLE
[ARTEMIS-1662] Reduce log of "INFO [org.apache.activemq.artemis.core.…

### PR DESCRIPTION
**Pull request by treblereel**
…client] (Thread-27 (ActiveMQ-client-global-threads))

jira: https://issues.apache.org/jira/browse/ARTEMIS-1662

### CODE REVIEW

File: artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java [hunk 1]
```diff
@@ -64,7 +64,7 @@
    @Message(id = 211001, value = "session created", format = Message.Format.MESSAGE_FORMAT)
    void dumpingSessionStack(@Cause Exception e);
 
-   @LogMessage(level = Logger.Level.INFO)
+   @LogMessage(level = Logger.Level.DEBUG)
```
Comment (michaelandrepearce) [inline artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java]: Client side this is very useful information, and should only occur on connection

Comment (treblereel) [reply artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java]: @michaelandrepearce ok, let me take a look

Comment (michaelandrepearce) [reply artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java]: Probably one option would be to make NettyConnector take a boolean value isServerConnection or something, then later 

      if (isServerConnection) {
         ActiveMQClientLogger.LOGGER.startedServerNettyConnector(connectorType,  TransportConstants.NETTY_VERSION, host, port);
      } else {
         ActiveMQClientLogger.LOGGER.startedNettyConnector(connectorType,  TransportConstants.NETTY_VERSION, host, port);
      }

Where the startedServerNettyConnector could log at DEBUG

Comment (treblereel) [reply artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java]: @michaelandrepearce, thanks, i am trying to do the same fix atm

Comment (michaelandrepearce) [reply artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java]: Agreed server side, but client side this is far too useful to lose.

Comment (clebertsuconic) [reply artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java]: @michaelandrepearce actually this is getting too verbose on clients.
We can use log.debug on the client... and you can increase logger if you really want it.

Some users will complain the opposite.

Comment (michaelandrepearce) [reply artemis-core-client/src/main/java/org/apache/activemq/artemis/core/client/ActiveMQClientLogger.java]: disagree still but as its 2 v 1 and its logging at the end of the day (bigger issues to really worry on) ill make it a -0. Ive merged the PR as is.

**Pull request closed without merging**